stallman/producao.md

```diff
@@ -1,4 +1,89 @@
 [Voltar](intro.md)
 
 Produção (livros, artigos, programas de computador, etc)
-====
\ No newline at end of file
+====
+# Livros publicados:
+
+### Free Software, Free Society: Selected Essays of Richard M. Stallman
```

Comment:
Colocar palavras estrangeiras em itálico, tipo _Free Software_

Reply:
Correção feita.